examples/twitter.amp.html

```diff
@@ -107,5 +105,17 @@ <h2>Timelines</h2>
       data-timeline-user-id="3450662892"
       data-tweet-limit="5">
   </amp-twitter>
+
+  <!-- BIND -->
+  <h2 on="tap:AMP.setState({tweetid: '949054860166553600'})">Change tweet</h2>
+  <amp-state id="tweetid">
+    <script type="application/json">638793490521001985</script>
+  </amp-state>
+  <amp-twitter width=375 height=472
+      layout="responsive"
+      data-tweetid="638793490521001985"
+      [data-tweetid]="tweetid"
+      >
```

Comment:
file convention thus far is to have closing `>` not on it's own line, but with the previous line.

Reply:
fixed 👍